python/katana/property_graph.pyx.jinja

```diff
@@ -39,16 +41,26 @@ cdef class PropertyGraph:
         :param node_properties: A list of node property names to load into memory. If this is None (default), then all properties are loaded.
         :param edge_properties: A list of edge property names to load into memory. If this is None (default), then all properties are loaded.
         """
-        if node_properties is not None or edge_properties is not None:
-            if node_properties is None or edge_properties is None:
-                raise ValueError("If either node_properties or edge_properties are provided, both must be provided.")
-            self.underlying = handle_result_value(
-                _PropertyGraph.MakeWithProperties(bytes(path, "utf-8"),
-                                                     _convert_string_list(node_properties),
-                                                     _convert_string_list(edge_properties)))
-        else:
-            self.underlying = handle_result_value(
-                _PropertyGraph.Make(bytes(path, "utf-8")))
+        cdef RDGLoadOptions opts
```

Comment:
If you can expose `partition_to_load` from python that would be great. We may need it eventually. The parameter should be `None` by default and `if partition_to_load is not None:` you can assign a value to the optional field in `opts`.